dotnet/src/Connectors/Connectors.Postgres.UnitTests/PostgresCollectionTests.cs

```diff
@@ -16,46 +13,6 @@ public class PostgresCollectionTests
 {
     private const string TestCollectionName = "testcollection";
 
-    private readonly Mock<IPostgresVectorStoreDbClient> _postgresClientMock;
-    private readonly CancellationToken _testCancellationToken = new(false);
-
-    public PostgresCollectionTests()
-    {
-        this._postgresClientMock = new Mock<IPostgresVectorStoreDbClient>(MockBehavior.Strict);
-        this._postgresClientMock.Setup(l => l.DatabaseName).Returns("TestDatabase");
-    }
-
-    [Fact]
-    public async Task CreatesCollectionForGenericModelAsync()
-    {
-        // Arrange
-        var recordDefinition = new VectorStoreRecordDefinition
-        {
-            Properties = [
-                new VectorStoreKeyProperty("HotelId", typeof(int)),
-                new VectorStoreDataProperty("HotelName", typeof(string)) { IsIndexed = true, IsFullTextIndexed = true },
-                new VectorStoreDataProperty("HotelCode", typeof(int)) { IsIndexed = true },
-                new VectorStoreDataProperty("ParkingIncluded", typeof(bool)) { IsIndexed = true, StoragePropertyName = "parking_is_included" },
-                new VectorStoreDataProperty("HotelRating", typeof(float)) { IsIndexed = true },
-                new VectorStoreDataProperty("Tags", typeof(List<string>)),
-                new VectorStoreDataProperty("Description", typeof(string)),
-                new VectorStoreVectorProperty("DescriptionEmbedding", typeof(ReadOnlyMemory<float>?), 100) { DistanceFunction = DistanceFunction.ManhattanDistance }
-            ]
-        };
-        var options = new PostgresCollectionOptions()
-        {
-            VectorStoreRecordDefinition = recordDefinition
-        };
-        var sut = new PostgresCollection<object, Dictionary<string, object?>>(this._postgresClientMock.Object, TestCollectionName, options);
-        this._postgresClientMock.Setup(x => x.DoesTableExistsAsync(TestCollectionName, this._testCancellationToken)).ReturnsAsync(false);
-
-        // Act
-        var exists = await sut.CollectionExistsAsync();
```

Comment:
this is covered by the integration tests, I don't see a reason to have internal abstraction just to verify that the public type returns what mock has been configured with